TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/api/implementations/objects/ProxyGroupObjectBasicImplementation.java

```diff
@@ -52,10 +52,10 @@
     private Collection<String> hostNames;
     @JsonProperty("jp")
     private List<String> javaParameters;
-    @JsonProperty("jdkp")
-    private String jdkPath;
+    @JsonProperty("jdrp")
```

Comment:
This seems to be a typo („jp“ would be sufficient, too)

Reply:
jp already used?